src/hevm/hevm.cabal

```diff
@@ -81,6 +81,7 @@ library
     ethjet/tinykeccak.h, ethjet/ethjet.h, ethjet/ethjet-ff.h, ethjet/blake2.h
   build-depends:
     QuickCheck                        >= 2.13.2 && < 2.15,
+    async,
```

Comment:
I think we need version bounds here for hackage releases?

Reply:
fixed